util/log/log.go

```diff
@@ -86,8 +86,8 @@ func Infof(format string, args ...interface{}) {
 
 // InfoDepth logs to the INFO log, offsetting the caller's stack frame by
 // 'depth'.
-func InfoDepth(depth int, args ...interface{}) {
-	logDepth(nil, depth+1, InfoLog, "", args)
+func InfoDepth(depth int, format string, args ...interface{}) {
```

Comment:
The lack of a `format` argument to these functions was deliberate: they are `Depth` variants of `log.Info`, not `log.Infof`.  If they take a format they should be renamed to `log.InfofDepth`.

The fact that our logging implementation supports an empty format string as equivalent to the non-`f` version of the logging function is slightly surprising (and inconsistent with the functions in package `fmt`), and either needs to be documented or we need to keep non-`f` versions of these logging functions for use in the raft log adapter.